packages/gatsby/src/utils/webpack.config.js

```diff
@@ -487,10 +477,10 @@ module.exports = async (
         // CSS modules
         config.loader(`cssModules`, {
           test: /\.module\.css$/,
-          loader: (stage === `build-html`
-            ? extractBuildHtml
-            : extractDevelopHtml
-          ).extract(`style`, [cssModulesConfig(stage), `postcss`]),
+          loader: extractTextPlugin(stage).extract(`style`, [
+            cssModulesConfig(stage),
+            `postcss`,
+          ]),
```

Comment:
The helper returns the right instance for both stages handled here, so the conditional is no longer needed.